Submitty/Submitty
# [Refactor:Router] New URL for PlagiarismController

### URL Changes

- GET `http://192.168.56.111/f19/sample/plagiarism`
- GET `http://192.168.56.111/f19/sample/plagiarism/check_refresh`
- GET `http://192.168.56.111/f19/sample/plagiarism/configuration/new`
- GET `http://192.168.56.111/f19/sample/plagiarism/configuration/edit`
- POST `http://192.168.56.111/f19/sample/plagiarism/configuration/new`
- GET `http://192.168.56.111/f19/sample/plagiarism/gradeable/<gradeable_id>`
- GET `http://192.168.56.111/f19/sample/plagiarism/gradeable/<gradeable_id>/rerun`
- GET `http://192.168.56.111/f19/sample/plagiarism/gradeable/<gradeable_id>/nightly_rerun`
- GET `http://192.168.56.111/f19/sample/plagiarism/gradeable/<gradeable_id>/match`
- GET `http://192.168.56.111/f19/sample/plagiarism/gradeable/<gradeable_id>/clicked_match`
- POST `http://192.168.56.111/f19/sample/plagiarism/gradeable/<gradeable_id>/delete`

### Testing
Manually tested the following:
- Create new config, edit that config, delete it
- Rerun and enables nightly rerun
- See matches 

I was unable to really see the code matches as every time I select two users instead of showing in parallel my chrome crashes... but I visited the URL and the content looks correct. Barb says the plagiarism controller isn't working now so it's probably because of that?
